### PR TITLE
INT-2332 handle if test, it, describe are already imported from other path/package

### PR DESCRIPTION
**Pull request by hbjORbj**

### CODE REVIEW

File: codemods/mocha/vitest/migrate-tests/src/index.ts [hunk 1]
```diff
@@ -40,6 +48,32 @@ export default function transform(
 		},
 	});
 
+	root.find(j.ImportDeclaration).forEach((importDeclaration) => {
+		importDeclaration.node.specifiers?.forEach(
+			(
+				specifier:
+					| ImportSpecifier
```
Comment (grzpab) [inline codemods/mocha/vitest/migrate-tests/src/index.ts:55]: why did you need to declare types?

Comment (grzpab) [reply codemods/mocha/vitest/migrate-tests/src/index.ts]: @hbjORbj the imports are still imported, linting has failed

Comment (hbjORbj) [reply codemods/mocha/vitest/migrate-tests/src/index.ts]: addressed

**Review by grzpab — APPROVED** [review]
